bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/service/SamsungTvUtils.java

```diff
@@ -68,12 +73,12 @@
 
     /**
      * Build {@link Document} from {@link String} which contains XML content.
-     * 
+     *
      * @param xml
-     *            {@link String} which contains XML content.
+     *                {@link String} which contains XML content.
      * @return {@link Document} or null if convert has failed.
```

Comment:
see below

Reply:
Sure, I reformatted all files with the ESH codestyle formatter properly enabled.